control/HTTPCacheControl.php

```diff
@@ -81,7 +81,10 @@ public function __construct()
 
 		// If we've not been provided an initial state, then grab HTTP.cache_contrpl from config
 		if (!$this->state) {
-			$this->setDirectivesFromArray(Config::inst()->get('HTTP', 'cache_control'));
+			$defaultDirectives = Config::inst()->get('HTTP', 'cache_control');
+			if (!empty($defaultDirectives)) {
+				$this->setDirectivesFromArray(Config::inst()->get('HTTP', 'cache_control'));
```

Comment:
Could we re-use `$defaultDirectives` here? Or even cast the config value as an array and skip the if statement?